themes/green/admin/assets/js/adjustments.js

```diff
@@ -193,8 +202,8 @@ function loadItems() {
 
         });
 
-        var col = 3;
-        var tfoot = '<tr id="tfoot" class="tfoot active"><th colspan="'+col+'">Total</th><th class="text-center">' + formatQty(parseFloat(count) - 1) + '</th>';
+        var col = 7;
+        var tfoot = '<tr id="tfoot" class="tfoot active"><th colspan="'+col+'">TotalM</th><th class="text-center">' + formatQty(parseFloat(count) - 1) + '</th>';
```

Comment:
What is TotalM ? Should be Total, right?

Reply:
Resolved